src/pages/AddPoll.tsx

```diff
@@ -0,0 +1,26 @@
+import Navbar from "../components/Navbar";
+import InputGroup from "react-bootstrap/InputGroup";
+import Form from "react-bootstrap/Form";
+
+const AddPoll: React.FC = () => {
+  return (
+    <>
+      <Navbar />
```

Comment:
navbar should not be here @Rishab-innovative 
if it's here then, you are going to add it to every page, so it should be global and imported once @Rishab-innovative